webpack/assets/javascripts/foreman_tools.js

```diff
@@ -80,3 +81,22 @@ export function initTypeAheadSelect(input) {
     width: '400px'
   });
 }
+
+// send get forms through turbolinks
+export function tableUpdated(element) {
```

Comment:
maybe name this more descriptively, something like searchTable or something? the name is a bit confusing